src/ipc-handlers.ts

```diff
@@ -55,6 +55,7 @@ import { DEFAULT_SITE_PATH, getResourcesPath, getSiteThumbnailPath } from 'src/s
 import { loadUserData, saveUserData } from 'src/storage/user-data';
 import { DEFAULT_PHP_VERSION } from 'vendor/wp-now/src/constants';
 import { openCertificate as openCertificateDialog } from './lib/certificate-manager';
+import { SupportedEditor } from './lib/editor';
```

Comment:
Should we pluralise the variable? 
```suggestion
import { SupportedEditors } from './lib/editor';
```

Reply:
Based on my personal opinion, we could keep singular here.

Reply:
I used singular case here following common TS patterns, and it also felt more natural as it's not an array or some collection.

Reply:
That makes sense. I think we should keep it as it is. 👍